common/formatting/layout_optimizer.cc

```diff
@@ -182,6 +188,21 @@ LayoutFunction::const_iterator LayoutFunction::AtOrToTheLeftOf(
   return it - 1;
 }
 
+LayoutFunction LayoutFunctionFactory::WrappedLine(
+    const UnwrappedLine& uwline) const {
+  const auto tokens = uwline.TokensRange();
+  auto token_lfs = absl::FixedArray<LayoutFunction>(tokens.size());
```

Comment:
Why? Generally I prefer this way of constructing because it always work (avoids Most Vexing Parse problem, never accidentally calls initializer_list constructor like `Type var{arg, ...};` could), and since C++17 there's guaranteed copy elision in such assignments.

Reply:
I guess both ways are fine; the `absl::FixedArray<LayoutFunction> token_lfs(tokens.size());` would be slightly quicker to parse for me as you never have to worry about if copy elision happens or not (but I guess that is only for me as I am working in various code-bases from c++03 to c++17). So I personally always would write it the way David suggested.

But with C++17 in mind, the assignment reads slighly more natural, I don't feel strongly either way. Looks good.